src/controller/insertcontent.js

```diff
@@ -50,7 +50,12 @@ export default function insertContent( dataController, content, selection, batch
 		isLast: true
 	} );
 
-	selection.setRanges( insertion.getSelectionRanges() );
+	const newRange = insertion.getSelectionRange();
+
+	/* istanbul ignore else */
```

Comment:
I'd like to have every case like this commented. We're not testing else because it's a safe check for some unpredictable edge cases. Also, having a `log.warning()` would be good there, like in the other cases in this method.